Signal/src/ViewControllers/PhotoLibrary/ImagePickerController.swift

```diff
@@ -12,21 +12,25 @@ protocol ImagePickerControllerDelegate {
 }
 
 @objc(OWSImagePickerGridController)
-class ImagePickerGridController: UICollectionViewController, PhotoLibraryDelegate {
+class ImagePickerGridController: UICollectionViewController, PhotoLibraryDelegate, PhotoCollectionPickerDelegate {
 
     @objc
     weak var delegate: ImagePickerControllerDelegate?
 
     private let library: PhotoLibrary = PhotoLibrary()
-    private let libraryAlbum: PhotoLibraryAlbum
-
-    var availableWidth: CGFloat = 0
+    private var photoCollection: PhotoCollection
```

Comment:
I've tried to standardize around the terms "photo/asset" and "photo collection" rather than "image" and "album", as I understand that your preference is to reflect the naming conventions of the Photos framework.

* `PhotoCollection` represents the metadata about a given collection.
* `PhotoCollectionContents` can be used to load the actual collection assets.  Renamed from `PhotoLibraryAlbum`.